internal/awslambda/extension/telemetry/api.go

```diff
@@ -0,0 +1,40 @@
+package telemetry
+
+import "github.com/atlassian/gostatsd/internal/awslambda/extension/api"
+
+const (
+	ApiVersion                              = "2022-07-01"
+	SubscribeEndpoint        api.LambdaPath = "/" + ApiVersion + "/telemetry"
+	ProtocolHTTP             string         = "HTTP"
+	PlatformSubscriptionType string         = "platform"
+	MinBufferingTimeoutMs    int            = 25
+
+	RuntimeDone EventType = "platform.runtimeDone"
+	// the sandbox hostname must be used in the lambda runtime, the port however can be configured if required
+	LambdaRuntimeAvailableAddr = "sandbox:8083"
+)
+
+type EventType string
+
+type Event struct {
+	// type is the only field we deserialise as it is used for flushing metrics
```

Comment:
Out of curiosity, is there any other values we'd care about?

Reply:
The full catalogue of events and their schemas can be found here https://docs.aws.amazon.com/lambda/latest/dg/telemetry-schema-reference.html#platform-initStart

We don't care about anything else apart from when the runtime is done to trigger the flush, that only requires the type field

Reply:
Easy, these might be useful in future to capture by default but we can save that for a future PR.

Reply:
I can link it the schemas here for reference in case they we ever need more.